### PR TITLE
Welcome GSoC 2021 Students!

### PR DESCRIPTION
**Pull request by ColinFinck**
Let's publicly welcome our GSoC 2021 students and also clarify why they were chosen and others not / how we choose students in general.

### CODE REVIEW

File: content/project-news/welcome-gsoc-2021-students.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Another year, another round of Google Summer of Code!
+The ReactOS Project is participating in Google Summer of Code 2021 and welcomes the selected students.
+
+**He Yang** is a successful student from GSoC 2020, who is going to continue his dive into user-mode ReactOS this year by working on _iernonce.dll_.
+That library is responsible for _RunOnceEx_, a prominent Windows feature to run multiple commands once after a system reboot.
+Implementing that feature will benefit software depending on it.
+Examples include installers that need to perform certain cleanup and completion tasks after the first reboot.  
+He Yang will be mentored by George Bișoc and Hermès Bélusca.
+
+**The_DarkFire_** has already impressed us with his [ReactOS live coding sessions](https://www.twitch.tv/the_darkfire_) prior to GSoC and proposed to complete his work on Symmetric Multiprocessing (SMP).
```
Comment (zefklop) [inline content/project-news/welcome-gsoc-2021-students.md:16]: `**_The_Darkfire_**` renders oddly. DarkFire is in italic, and all underscores are eaten.

Comment (learn-more) [inline content/project-news/welcome-gsoc-2021-students.md:16]:
```suggestion
**The\_DarkFire\_** has already impressed us with his [ReactOS live coding sessions](https://www.twitch.tv/the_darkfire_) prior to GSoC and proposed to complete his work on Symmetric Multiprocessing (SMP).
```

File: content/project-news/welcome-gsoc-2021-students.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+An SMP-aware Hardware Abstraction Layer (HAL) enables ReactOS to use more than a single processor core, greatly enhancing its usability on modern hardware.
+Once completed, we also expect this work to trigger a lot of concurrency bugs still hidden in ReactOS.
+Hence, finding and fixing these bugs will also improve the overall experience even for single-core machines.  
+The_DarkFire_ will be mentored by Victor Perevertkin and Jérôme Gardou.
```
Comment (learn-more) [inline content/project-news/welcome-gsoc-2021-students.md:20]:
```suggestion
The\_DarkFire\_ will be mentored by Victor Perevertkin and Jérôme Gardou.
```

**Review by learn-more — APPROVED** [review]
Other than the escaping issues, lgtm

File: content/project-news/welcome-gsoc-2021-students.md (hunk 1 above)
Comment (DarkFire01) [reply content/project-news/welcome-gsoc-2021-students.md]: I would like to officially state I am 100% okay with just my real name (Justin Miller) being used.
For one thing any PRs I do in this project have to use my real name anyway, the fact I used my nick in my GSoC was an accident too.

Comment (ColinFinck) [reply content/project-news/welcome-gsoc-2021-students.md]: Thanks for the clarification! Will change accordingly

**Review by HBelusca — COMMENTED** [review]
nice text!

**Pull request merged**